backend/uwsgi.ini

```diff
@@ -0,0 +1,12 @@
+[uwsgi]
```

Comment:
This is a configuration file for the uwsgi server. https://uwsgi-docs.readthedocs.io/en/latest/WSGIquickstart.html